src/main/java/io/github/pylonmc/pylon/base/content/machines/fluid/FluidStrainer.java

```diff
@@ -130,6 +130,12 @@ public void onFluidRemoved(@NotNull PylonFluid fluid, double amount) {
         ));
     }
 
+    @Override
+    @NotNull
+    public Component getGuiTitle() {
+        return Component.translatable("pylon.pylonbase.item.fluid_strainer.name");
```

Comment:
```suggestion
        return getDefaultName();
```
Or whatever the function is called